### DISCUSSION TITLE
How can I disable predefined keys(maybe it's plugin or not) due to  configure Treesitter Text object setting ( selection)

### DISCUSSION THREAD

**Discussion by Cruising0904** [LazyVim/LazyVim#585] [Q&A] [answered]
It's been Three whole days to migrate from personal neovim config to lazyvim. Highly satisfied with performance and features. Now just few things to do. 

I managed to map object move and repeatable, but can't with selection

How can I disable some of these keys? I am sure which plugin that they are used maybe it's just predefined keys 
Where can I check where they are configured?  maybe it's just plugin I have no clue 

![image](https://user-images.githubusercontent.com/29267295/232112046-c22fd252-5a4c-44bc-80a8-043793c75a61.png)
I want to disable these keys 

![image](https://user-images.githubusercontent.com/29267295/232112701-946d69c6-daac-4d16-aefa-fb92b0cdde68.png)
This is jump motion I mapped.(Treesitter-Text-object

![image](https://user-images.githubusercontent.com/29267295/232112812-8013f15b-2e2d-45d9-b39e-6d913be944a4.png)
I configured like this in treesitter plugin file

looked through many times to find a way if it was overwrote, I can't seem to find.

Please help appreciate it

**Comment by amaanq** [comment]
This requires nvim-treesitter-textobjects, ensure that's loaded. The mapping are also for **visual** mode, ensure you're in that as well. Also, not every language is supported in nvim-ts-textobjects, let alone nvim-ts, but all the big/common ones are.

Also ensure the select table is inside the textobjects module table for nvim-ts

**Reply by Cruising0904** [reply]
Man I already mapped treesitter-text-object and it's loaded for sure. One of the features that I can't map is selection (something like viw ciw commands) move is already set. :)

**Comment by Cruising0904** [comment]
I think I found it! 
it's Mini.ai plugin I will close the issue

**Reply by amaanq** [reply]
nice, glad you figured it out

**Reply by Cruising0904** [reply]
Thank you :-) but it still doesn't work though

**Comment by Cruising0904** [comment, marked as answer]
Disabled mini.ai 
Now when I press 'vi' command look like this
![image](https://user-images.githubusercontent.com/29267295/232180160-070b9639-6f2f-444c-b944-d05d6ba0685f.png)

and my configuration does not show to it

**Reply by Cruising0904** [reply]
after re-trey few times it works now
